cterasdk/core/cloudfs.py

```diff
@@ -178,7 +178,7 @@ def find(self, name, owner, include=None):
             if cloud_folder.owner.endswith(owner_ref):
                 return cloud_folder
 
-        logging.getLogger().info('Could not find cloud folder. %s', {'folder': name, 'owner': owner})
+        logging.getLogger().info('Could not find cloud folder. %s', {'folder': name, 'owner': str(owner)})
         raise CTERAException('Could not find cloud folder', None, folder=name, owner=owner)
```

Comment:
Shouldn't you do the same for the exception?

Reply:
Correct. I also updated the implementation to use a query filter instead of string comparison